cassandra-examples/src/main/java/io/vertx/cassandra/prepared/PreparedExample.java

```diff
@@ -0,0 +1,43 @@
+package io.vertx.cassandra.prepared;
+
+import com.datastax.driver.core.PreparedStatement;
+import io.vertx.cassandra.CassandraClient;
+import io.vertx.cassandra.CassandraClientOptions;
+import io.vertx.core.AbstractVerticle;
+import io.vertx.core.Launcher;
+
+public class PreparedExample extends AbstractVerticle {
+
+  /**
+   * Convenience method so you can run it in your IDE
+   */
+  public static void main(String[] args) {
+    Launcher.main(new String[]{"run", PreparedExample.class.getName()});
+  }
+
+  @Override
+  public void start() {
+    CassandraClient client = CassandraClient.createNonShared(vertx, new CassandraClientOptions().setPort(9042));
```

Comment:
It would be better to use `createShared`, because many users start copy/pasting example code. And usually you want a single `Session`.